tribe-ext-map-show-all-events-with-same-venue/index.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar PRO Extension: Displays multiple events with the same venue in Google Maps infoWindow.
```

Comment:
I wonder if it something like _"Google Maps tooltips"_ might be more widely understood?